components/BottomNavigation/README.md

```diff
@@ -41,6 +42,12 @@ navigation icon takes you directly to the associated view or refreshes the curre
 - [Extensions](#extensions)
   - [Color Theming](#color-theming)
   - [Typography Theming](#typography-theming)
+- [Accessibility](#accessibility)
+  - [`-accessibilityLabel`](#`-accessibilitylabel`)
+  - [Set `-accessibilityValue` ](#set-`-accessibilityvalue`-)
+  - [Good](#good)
```

Comment:
I'm not sure we want these as headings.

Reply:
The "Good" and "Bad" headings are still here. I still believe we can roll them into a section on `accessibiltyLabel`.  It might make sense to have a section for each property. If users have to touch multiple classes (`MDCBottomNavigationBar` as well as `UITabBarItem`), we can break those out into supersections.